src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/MethodWithGCInfo.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
-
+using System.Linq;
```

Comment:
Nit - is the <code>System.Linq</code> namespace really used in this file? I know that VS adds <code>using</code> clauses automatically when editing, it's just that at some point we'll likely want to audit Crossgen2 code using Linq for performance reasons and the superfluous <code>using</code> clause might add undesirable extra noise to the scan.

Reply:
It is needed for the existing `SequenceEqual` calls.